src/mpyl/cli/build.py

```diff
@@ -390,8 +450,17 @@ def jenkins(  # pylint: disable=too-many-arguments
         pipeline_parameters = (
             {"TEST": "true" if test else "false", "VERSION": version} if version else {}
         )
+
+        pipeline_parameters["BUILD_PARAMS"] = ""
+
+        if dryrun_:
+            pipeline_parameters["BUILD_PARAMS"] += " --dryrun"
+        if all_:
+            pipeline_parameters["BUILD_PARAMS"] += " --all"
         if arguments:
             pipeline_parameters["BUILD_PARAMS"] = " ".join(arguments)
```

Comment:
this would overwrite the dry_run and all params again btw

Reply:
I'm saying that if someone passes `arguments` you are ignoring all the other params because of `" ".join`

Reply:
Ah yes, nice spot. I've put the `" ".join` on top